Firebase/Messaging/FIRMessagingRmq2PersistentStore.m

```diff
@@ -102,6 +102,14 @@
 // table data handlers
 typedef void(^FCMOutgoingRmqMessagesTableHandler)(int64_t rmqId, int8_t tag, NSData *data);
 
+// Utility to create an NSString from a sqlite3 result code
+NSString * _Nonnull FIRMessagingStringFromSQLiteResult(int result) {
+  const char *errorStr = sqlite3_errstr(result);
+  //NSString *errorString = [NSString stringWithCString:errorStr encoding:NSUTF8StringEncoding];
```

Comment:
This commented line can probably be removed. But out of curiosity, why use a format instead of creating an NSString from the C string?

Reply:
Nice catch! Commented-out line removed. The main reason for going with the format was because I want to show it as "`resultCode` - `errorMsg`". Using `initWithCString` I'd end up creating a string with format anyway.